charts/index.php

```diff
@@ -196,6 +196,7 @@
 	function changePage(newPage) {
 		page = Math.min(Math.max(newPage, 1), 9);
 		updateChart();
+		window.scrollTo(0,0);
```

Comment:
I think the UX of this would feel a lot better if the window scrolled when the state of the page changed i.e. in `resetPaginationDisplay()`. Right now it feels a little early?

Reply:
Yeah, I changed the code. Also gave it the "smooth" property so it goes to the top with an animation instead of jumping immediately.